docs/kitematic/create_pr.md

```diff
@@ -16,17 +16,15 @@ Rebasing helps verify that your commit is clean, and allows for an easier and fa
 
 Once you've cleaned up your commits:
 
-1. Push your code to your repository under the same name as your local branch.
+1. Push your code to your repository under the same name as your local branch (in the case of this example, 1191-branch).
 
-        $ git push origin xxxx-branch:xxxx-branch
-
-      where _xxxx_ is the number of the issue your worked on.
+        $ git push origin 1191-branch
```

Comment:
this needs to be `git push origin 1191-branch:1191-branch` in order to create the remote branch with the same name as the local branch. 
The above only works if the remote branch exists and has the same name